connexion/apis/aiohttp_api.py

```diff
@@ -293,13 +293,50 @@ async def get_request(cls, req):
         :rtype: ConnexionRequest
         """
         url = str(req.url)
-        logger.debug('Getting data and status code',
-                     extra={'has_body': req.has_body, 'url': url})
+
+        logger.debug(
+            'Getting data and status code',
+            extra={
+                # has_body | can_read_body report if
+                # body has been read or not
+                # body_exists refers to underlying stream of data
+                'body_exists': req.body_exists,
+                'can_read_body': req.can_read_body,
+                'content_type': req.content_type,
+                'url': url,
+            },
+        )
 
         query = parse_qs(req.rel_url.query_string)
         headers = req.headers
         body = None
-        if req.body_exists:
+
+        # if request is not multipart, `data` will be empty dict
```

Comment:
oh, hey :D that's mine :D

Reply:
Uh yeah.. much of this is yours ;)  ... including the funny-formatted unit tests above.  I started with your PR.  I thought I said that somewhere.  Maybe it's in my internal notes or a commit msg somewhere.

Anyway, it was much helpful in getting started.